Source/WebCore/platform/graphics/cocoa/controls/ApplePayButtonCocoa.mm

```diff
@@ -48,14 +99,22 @@
         borderRect.radii().bottomRight().maxDimension()
     });
 
-    auto systemImage = ApplePayButtonSystemImage::create(
-        owningApplePayButtonPart().buttonType(),
-        owningApplePayButtonPart().buttonStyle(),
-        owningApplePayButtonPart().locale(),
-        largestCornerRadius
-    );
+    GraphicsContextStateSaver stateSaver(context);
 
-    context.drawSystemImage(systemImage, borderRect.rect());
+    context.setShouldSmoothFonts(true);
+    context.scale(FloatSize(1, -1));
+
+    auto logicalRect = borderRect.rect();
+    
+    PKDrawApplePayButtonWithCornerRadius(
+        context.platformContext(),
+        CGRectMake(logicalRect.x(), -logicalRect.maxY(), logicalRect.width(), logicalRect.height()),
+        1.0,
+        largestCornerRadius,
+        toPKPaymentButtonType(owningApplePayButtonPart().buttonType()),
```

Comment:
Nit: Consider storing `owningApplePayButtonPart()` in a local variable, since it's used 3 times.